core/src/main/scala/org/apache/spark/h2o/converters/ConverterUtils.scala

```diff
@@ -28,7 +28,8 @@ import scala.collection.mutable.ListBuffer
 import scala.reflect.runtime.universe._
 
 
-private[converters] trait ConverterUtils {
+//private[converters] - will uncomment when H2OContext moves into converters
```

Comment:
Why should be H2OContext in converters package ? 

Convertors package contains all supported types from which we can transfer to `H2OFrame` and back. `H2OContext` is like orchestrator of communication inside sparkling-water, from my point of view it does not belong to `convertors` and `ConvertorUtils` should remain private to `converters`